### PR TITLE
Create example for adding env variables from next.js.config / Final

### PR DESCRIPTION
**Pull request by pkellner**
This is the work in progress of creating env from next.js.config. Once "we" figure out why it's not working, I'll update the pull request to make it complete including README

### CODE REVIEW

**Comment by timneutkens** [comment]
Can you run `yarn lint-fix

**Comment by pkellner** [comment]
I’m out for a couple hours but will do when home.

**Comment by timneutkens** [comment]
@pkellner I've pushed up a commit, you were returning `env` instead of an object (the next.config.js object) holding the `env` key.

File: examples/with-env-from-next-config-js/.eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+module.exports = {
```
Comment (timneutkens) [inline examples/with-env-from-next-config-js/.eslintrc.js]: I don't think this is needed.

**Review by timneutkens — APPROVED** [review]

**Pull request merged**